templates/files/k8s-resource/aws-cni.yaml

```diff
@@ -140,10 +140,6 @@ spec:
             - name: MINIMUM_IP_TARGET
               value: "2"
             ## Deviation from original manifest - 5
-            ## disable SNAT as we setup NATGW in the route tables
-            - name: AWS_VPC_K8S_CNI_EXTERNALSNAT
-              value: "true"
```

Comment:
I would rather set the value to `false` explicitly here.

Reply:
Done